inventory/sample-ha.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,56 @@
+# 'hosting_infrastructure' is used to drive the correct behavior based
+# on the hosting infrastructure, cloud provider, etc. Valid values are:
+# - 'openstack'
+# - 'aws' (Coming Soon)
+# - 'azure' (Coming Soon)
+hosting_infrastructure: aws
+
+# AWS Provisioning Variables
+# use the "-e" option to set "aws_ssh_public_key"
+aws_image_name: <ec2 AMI>
+aws_num_masters: 3
+aws_num_nodes: 5
+aws_num_infra: 3
+aws_access_key: <Access key ID>
```

Comment:
Could we just load these up from environment variables rather than having them set in the inventory? I.e.: something like: `"{{ lookup('env','AWS_ACCESS_KEY') }}"`

Reply:
Sure, that will remove the requirement of setting this values in ec2.ini as well.